server/sonar-web/src/main/js/components/source-viewer/header.js

```diff
@@ -75,11 +75,11 @@ export default Marionette.ItemView.extend({
     if (this.options.viewer.highlightedLine) {
       query = query + '&line=' + this.options.viewer.highlightedLine;
     }
-    window.open('/component/index?' + query, this.model.get('name'), windowParams);
+    window.open(window.baseUrl + '/component/index?' + query, this.model.get('name'), windowParams);
   },
 
   showRawSources () {
-    const url = '/api/sources/raw?key=' + encodeURIComponent(this.model.get('key'));
+    const url = window.baseUrl + '/api/sources/raw?key=' + encodeURIComponent(this.model.get('key'));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)